src/libraries/Common/src/Interop/Windows/BCrypt/Interop.Blobs.cs

```diff
@@ -235,6 +243,18 @@ internal struct BCRYPT_ECCFULLKEY_BLOB
             // The rest of the buffer contains the domain parameters
         }
 
+        /// <summary>
+        ///     Used as a header to PQC parameters including the parameters set and key/seed.
+        /// </summary>
+        [StructLayout(LayoutKind.Explicit, Pack = 1)]
+        internal struct BCRYPT_PQDSA_KEY_BLOB
+        {
+            [FieldOffset(0)] internal KeyBlobMagicNumber Magic;
+            [FieldOffset(4)] internal int cbParameterSet;           // Byte size of parameterSet[]
+            [FieldOffset(8)] internal int cbKey;                    // Byte size of key[]
```

Comment:
The offsets/packing/padding here need to match what native things on Windows will do, since it's Win32 that will start reading at (blob + sizeof(BCRYPT_PQDSA_KEY_BLOB)).

.NET Sequential packing will do that; so just go ahead and change back to a normal looking struct.

Reply:
In this case it will but I shared an example in another comment where it won't. I think it's better to be explicit. The pack=1 ensures no trailing padding for the struct. But that would also affect padding between fields so we'd want explicit layout to make sure we match C/Win32.

Reply:
We use `sizeof` so if we want that to give the value without trailing padding, then we need the `Pack = 1`. I'm assuming Windows also ignores trailing padding when determining the start point to write the dynamic arrays?

Reply:
The only times it's wrong are when using .NET `bool` for Win32 `BOOL`, because they're not the same type (.NET is "I1", and Win32 is "I4").  The marshaller will project them when passing a struct by value, or when using PtrToStructure.  But so long as the `bool` vs `BOOL` doesn't come up, Sequential will produce the same offsets as the managed and native sides.

And "are the fields in the right order using Sequential" is much easier to read than "are all the offsets correct in both 32 bit and 64 bit with Explicit"? (Since there are no pointers here, "yes", but still).